e2etests/features/eyfs_p1_communication_and_language.feature

```diff
@@ -11,8 +11,8 @@ Feature: EYFS - Communication and Language - Links
     Then page has heading "Communication and language"
     Then "Left menu pane" item "1" is "Communication and language"
     And the following "links" are available:
-      | Interactions[3 times]                |
```

Comment:
Has this been deleted intentionally?

Reply:
It has not been deleted

It looks for the Interactions link three times